security/publish_spec.rb

```diff
@@ -3,9 +3,20 @@
 describe 'Publish', type: :feature do
   before :each do
     @conf = read_config
+    @aem_publish = init_publish_client(@conf['aem']['publish'])
   end
 
   it 'should not be able to login using default admin password' do
+    # use aem api calls
+    bundle = @aem_publish.bundle('com.adobe.cq.social.cq-social-forum')
+    begin
+      result = bundle.start
+    rescue RubyAem::Error => error
+      # response should be unauthorized
```

Comment:
Remove this, and rename the test to be "should not be able to start bundle" , let's figure out another way to test login using default admin password security.